release/build_release_package.sh

```diff
@@ -5,6 +5,7 @@ rm -f release/magento1702.zip
 rm -rf release/magento
 rm -rf vendor
 
+composer install -vvv --no-dev -d src/lib/Shopgate/cloud-integration-magento-oauth2
```

Comment:
was this not happening before? When I look at previous release versions, they seem to have the vendor folder for this lib. But when I look at it, I don't really know how it worked. You experienced some issues?

Reply:
So yes, I am still in a quandary why it is needed, especially when we already have this in the travis load script: https://github.com/shopgate/connect-integration-magento/blob/PI-9303-App-only-coupons-do-not-apply-if-you-register-in-webcheckout/.travis.yml#L8